dotnet/src/Connectors/Connectors.OpenAIV2/Connectors.OpenAIV2.csproj

```diff
@@ -13,6 +13,7 @@
   <!-- IMPORT NUGET PACKAGE SHARED PROPERTIES -->
   <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
   <Import Project="$(RepoRoot)/dotnet/src/InternalUtilities/src/InternalUtilities.props" />
+  <Import Project="$(RepoRoot)/dotnet/src/InternalUtilities/openai/OpenAIUtilities.props" />
```

Comment:
`openai` folder is not visible in .NET solution, we should probably update `.sln` file to include it. 
I'm also wondering, if we should create a dedicated `openai` folder and `.props` file comparing to adding `OpenAI`-related utilities under `src/OpenAI` folder. I guess the disadvantage of this approach is that we will include `OpenAI` utilities to every assembly that imports `InternalUtilities`, even in assemblies that don't use `OpenAI` functionality, but I think we need to fix the way how we include `InternalUtilities` in each assembly. Instead of importing entire folder, we should probably import only the classes that are actually used in specific assembly, but not all of them. This will allow us to keep all internal utilities under one `src` folder.

Reply:
Good point on the folder. Going to add it to the solution.

In this approach e are not creating OpenAI for every, as you can see in the line above the InternalUtilities for general projects are in the sibling `src` folder.